FWCore/Utilities/interface/stemFromPath.h

```diff
@@ -10,7 +10,7 @@ namespace edm {
   // object lives long enough.
   //
   // The reason to have our own function instead of
-  // std/boost::filesystem is that tehcnically these paths are not
+  // std/std::filesystem is that tehcnically these paths are not
```

Comment:
No need to repeat `std` in the comment
```suggestion
  // std::filesystem is that tehcnically these paths are not
```

Reply:
Thanks!